vulcanai2/tests/models/conftest.py

```diff
@@ -223,13 +223,13 @@ def multi_input_cnn_add_input_network(conv1D_net, conv2D_net,
         },
         device='cpu'
     )
-    net.add_input_network(conv3D_net)
```

Comment:
you shouldn't add an input after initiating the network hence why it is a private function

Reply:
makes sense! yeah